pkg/asset/agent/installconfig.go

```diff
@@ -215,9 +215,9 @@ func (a *OptionalInstallConfig) validateControlPlaneConfiguration(installConfig
 	var fieldPath *field.Path
 
 	if installConfig.ControlPlane != nil {
-		if *installConfig.ControlPlane.Replicas != 1 && *installConfig.ControlPlane.Replicas != 3 {
+		if *installConfig.ControlPlane.Replicas < 1 || *installConfig.ControlPlane.Replicas > 5 || *installConfig.ControlPlane.Replicas == 2 {
```

Comment:
(and very likely we'll have to remove the `2` case after https://github.com/openshift/installer/pull/9159 will land)